### ISSUE TITLE
Bug with xml2tlk

### ISSUE THREAD

**Issue by Aqvilinus** [xoreos/xoreos-tools#52]
I'm getting a strange error when trying to use it with **--dragonage** and **--version40** flags together on the file that was previously unpacked with **tlk2xml** tool. 

If you need an example of tlk, let me know.

**Comment by DrMcCoy** [comment]
Hmm, can you describe what those "strange errors" are? What exactly is happening vs. what did you expect to happen?

Keep in mind that currently, the xml2tlk tool can't create files readable by the Dragon Age games (or Sonic Chronicles), only by the older games. (As such, the `--dragonage` flag is maybe not all that useful).

I.e. what's currently only supported is to either take a TLK from, say, NWN, convert it into a XML, modify it and convert it back to be read by NWN. Or take a TLK from Dragon Age, convert it into a XML, modify it and convert it back to be read by **NWN** (or any of the other games), not Dragon Age. You'll also need to use the `--nwn` flag for that.

If that second case isn't working, that's something I can investigate in the evening.

(The reason is that the TLKs in the Dragon Age games are GFFs, and that xml2tlk tool preceded our GFF writer, and IIRC we don't even have a GFF4 writer anyway yet. Previous games had their own binary format for TLK, and the `--version40` flag is about that version number, not the GFF version.)

TL;DR: If you're trying to create a file the Dragon Age games can read, this tool can't do that yet, sorry.

**Comment by DrMcCoy** [comment]
Closing, feel free to reply and add more information if you think this is still an issue.
